src/layouts/Form/FieldWrapper.js

```diff
@@ -68,6 +69,10 @@ class FieldWrapper extends React.Component {
     error: null,
   };
 
+  componentDidMount() {
+    warning(true, 'DEPRECATED: the Field system is now deprecated. Please use Flow instead. (from FieldWrapper)');
```

Comment:
FYI: May want to consider using an HOC, this would allow you to deprecate a component either within the component or prior to being exported by the library, this would allow you to do things like build a helper to deprecate components (i.e. you could have a helper that reads from read from a JSON file and automatically deprecates components prior to export)